python/tests/test_likelihood.py

```diff
@@ -38,61 +70,45 @@ def test_lz_derivative_1_output_scalar():
 
 
 def test_one_dim_newton_rhapson():
-    def f(x):
+    @numba.njit
```

Comment:
I might be wrong, but I don't think numba will really do anything for a function that is only numpy calls. Those are already basically compiled down AFAIK

Reply:
You are absolutely right. The reason I did this is because the function `minimize_newton_rhaphson` is numbified with `njit` which is equivalent to `jit(nopython=True`. I cannot this function a python function as argument because it cannot run it.

Reply:
Ah, I see. Makes sense, thanks